removed ovmsclient (#3620)
openvinotoolkit/model_server: docs/clients_tfs.md

## Python Client

> When creating a Python-based client application, there is the package on PyPi that can be used with OpenVINO Model Server:
> - [tensorflow-serving-api](https://pypi.org/project/tensorflow-serving-api/)

Reviewer: ```suggestion
When creating a Python-based client application, you can use [tensorflow-serving-api](https://pypi.org/project/tensorflow-serving-api/) package that can be used with OpenVINO Model Server:
```
